src/app/modules/assets/list/asset-list.component.ts

```diff
@@ -338,7 +338,7 @@ export class AssetListComponent implements OnInit, OnDestroy {
       {
         icon: 'fa-ban',
         massAction: false,
-        label: 'ASSET_ACTIONS.TERMINATE',
+        label: 'Terminate',
```

Comment:
Why did we remove picking label from translation file ?

Reply:
changes added.

Reply:
![Screenshot 2024-06-04 124421](https://github.com/congarevenuecloud/e-commerce/assets/154309114/ce0aa113-50b8-47e8-b116-95de59d1921e)